### ISSUE TITLE
schematron accepts cac: DeliveryAddress

### ISSUE THREAD

**Issue by petredanoiu** [phax/validation#315]
According to CEN / TS 16931-3-2
Delivery address information is grouped in / Invoice / cac: Delivery / cac: DeliveryLocation.
However, the schematron validator accepts too and cac: DeliveryAddress.
Is this a weakness that should be corrected? 

<param name="UBL-CR-xxx" value="not(cac:Delivery/cac:DeliveryAddress)"/>
<assert test="$UBL-CR-xxx" flag="warning" id="UBL-CR-xxx">[UBL-CR-xxx]-A UBL invoice should not include the Delivery DeliveryAddress</assert>

**Comment by phax** [comment]
Thanks for the separate issue - much easier to find any maintain :)

There is an assertion `UBL-CR-682` of level warning that should deal with `DeliveryAddress`:
```xpath
not(cac:Delivery/cac:DeliveryAddress)
```
It is execute in a rule in context
```
<rule context="/ubl:Invoice | /cn:CreditNote">
```

Can you post an example file here?

**Comment by petredanoiu** [comment]
indeed
this problem was solved in 1.3.8 release
10x!

**Comment by phax** [comment]
Okidoki :)
